evdns.c

```diff
@@ -416,6 +416,10 @@ struct evdns_base {
 #endif
 
 	int disable_when_inactive;
+
+  // used when nameserver probe failed
+  int ns_max_prob_timeout;
+  int ns_timout_backoff_factor;
```

Comment:
Should indent with tabs here. (Done correctly in the rest of the changeset, looks like a small oversight.)

Why omit the "e" from "probe" for the option names?

Reply:
I'll fix the indent. Sorry for the bad naming, will add the `e` letter.